ramls/isbn.raml

```diff
@@ -47,11 +46,8 @@ types:
         400:
           description: "Bad request, an invalid ISBNs is provided"
           body:
-            application/json:
-              type: errors
-              example:
-                strict: false
-                value: !include raml-util/examples/errors.sample
+            text/plain:
+              example: "ISBN value is invalid"
```

Comment:
Looks like changing response from json to plane text is breaking change and should be reflected in raml and module descriptor versions.

Reply:
Done, increased the major version for "isbn-utils" interface